### ISSUE TITLE
[FORMATTING] var := value Token for Snowflake is Parsed as var: = value.

### ISSUE THREAD

**Issue by chiefupstart** [sql-formatter-org/sql-formatter#775]
**Input SQL**
```sql
DECLARE
 profit NUMBER(38, 2);
 revenue NUMBER(38, 2);
 cost NUMBER(38, 2);
BEGIN
 let profit := revenue - cost;
RETURN profit;
```
**Expected Output**
I would expect sql-formatter to honor the := symbol for Snowflake.

**Actual Output**
```
Parse error at token: = at line 6 column 13 <-- notice the space in the token.
Unexpected OPERATOR token: {"type":"OPERATOR","raw":"=","text":"=","start":91}. Instead, I was expecting to see one of the following:
A LINE_COMMENT token based on: ...
A BLOCK_COMMENT token based on: ...
A DISABLE_COMMENT token based on: ...
Etc etc.
```
**Usage**
```js
      let serverLanguage: SqlLanguage = 'snowflake';
      formattedSql = format(sqlText, {
        language: serverLanguage,
        tabWidth: 2,
        keywordCase: 'upper',
        linesBetweenQueries: 1,
      });
```

Version: "sql-formatter": "^15.4.1"

**Comment by nene** [comment]
Thanks for reporting.

**Comment by nene** [comment]
Fixed in 15.4.2
